### PR TITLE
enchancement: Added archival meetings display

### LINKED ISSUE #41: [Enhancement]: Add archival meetings and add text to display when there arent any upcoming meetings to `gwardia/index`
### My idea

My proposition is to add button at bottom of meetings link, that shows all archival meetings when clicked
And for no meetings just meeting container with text saying that there ar eno upcoming meetings

### PR DESCRIPTION
**Pull request by Sanielia**
It can be toggled by button.
Also I have made index site display all upcoming meetings, not only five next

### CODE REVIEW

**Review by tapczanek1321 — APPROVED** [review]

File: core/templates/core/base.html [hunk 1]
```diff
@@ -7,7 +7,7 @@
         <title>GwardiaHub | {% block title %}{% endblock %}</title>
         <script src="https://cdn.tailwindcss.com"></script>
         {% load static %}
-        <script src="{% static "tailwind_custom_styles.js" %}"></script>
+        <script src="{% static 'GwardiaHub/tailwind_custom_styles.js' %}"></script>
```
Comment (Norbiros) [inline core/templates/core/base.html]: Why? And I think we should use double quotes here...

File: gwardia/templates/gwardia/index.html [hunk 2]
```diff
@@ -6,23 +6,57 @@
...
+                        </li>
+                    {% endfor %}
+                {% else %}
+                    <li class="flex flex-col items-center border-solid border-gray-500 border-2 p-5 w-full mb-1 rounded-2xl">
+                        <p class="w-full text-center">Brak nadchodzących spotkań</p>
+                    </li>
+                {% endif %}
+            </ul>
+        </div>
+        <div class="w-1/2 hidden" id="js-archival-meetings-container">
```
Comment (Norbiros) [inline gwardia/templates/gwardia/index.html]: Now we have same code copied 2 times (except archival meetings don't have nice description wgen list is empty). Maybe extract it to component ([inclusion tag](https://docs.djangoproject.com/en/dev/howto/custom-template-tags/#inclusion-tags))

**Review by Norbiros — CHANGES_REQUESTED** [review]

File: core/templates/core/base.html (hunk 1 above)
Comment (Sanielia) [reply core/templates/core/base.html]: image

File: gwardia/templates/gwardia/list_events.html [hunk 3]
```diff
@@ -0,0 +1,16 @@
+{% for event in events %}
+    {% if event.is_archival %}
```
Comment (Norbiros) [inline gwardia/templates/gwardia/list_events.html]: Indent! If we add it in `for`, why you are not doing it in `if`?

File: gwardia/templates/gwardia/list_events.html [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+    <li class="js-meeting-container flex flex-col items-center border-solid border-gray-500 border-2 p-5 w-full mb-1 rounded-2xl cursor-pointer bg-orange-300 bg-opacity-25">
```
Comment (Norbiros) [inline gwardia/templates/gwardia/list_events.html]: `<li>` is copied 2 times, why?

File: gwardia/templates/gwardia/index.html [hunk 5]
```diff
@@ -2,27 +2,42 @@
...
-                    <p class="js-meeting-description border-gray-500 p-5 w-full break-all border-t-2 hidden">
-                        {{ meeting.description|linebreaksbr }}
-                    </p>
-                </li>
-            {% endfor %}
-        </ul>
+        <div  class="w-1/2" id="js-next-meetings-container">
+            <h2 class="text-2xl text-white text-center font-bold mt-8 mb-4">Nadchodzące spotkania</h2>
+            <ul>
+                {% if upcoming_meetings %}
```
Comment (Norbiros) [inline gwardia/templates/gwardia/index.html]: Maybe you could move this `if` to `list_events` and maybe also `h2`?

**Review by Norbiros — CHANGES_REQUESTED** [review]

File: gwardia/templates/gwardia/list_events.html (hunk 4 above)
Comment (Sanielia) [reply gwardia/templates/gwardia/list_events.html]: one have changed background (archived have orange background)

File: gwardia/templates/gwardia/index.html (hunk 5 above)
Comment (Sanielia) [reply gwardia/templates/gwardia/index.html]: If we want to use this as event list, not meetings list, then we shouldn't use h2 with text specific to meetings
Maybe I misunderstood you intentions and it was supposed to be meetings list tag from start

File: gwardia/templates/gwardia/list_events.html (hunk 4 above)
Comment (Norbiros) [reply gwardia/templates/gwardia/list_events.html]: Hmmm. Maybe add something like that `{% if event.is_archival %}bg-orange-300 bg-opacity-25{% endif %}`...

File: gwardia/templates/gwardia/index.html [hunk 6]
```diff
@@ -2,27 +2,30 @@
...
-                        <span class="break-all">{{ meeting.title }}</span>
-                        <span class="float-right">{{ meeting.date_of_meeting }}</span>
-                    </p>
-                    <p class="js-meeting-description border-gray-500 p-5 w-full break-all border-t-2 hidden">
-                        {{ meeting.description|linebreaksbr }}
-                    </p>
-                </li>
-            {% endfor %}
-        </ul>
+        <div  class="w-1/2" id="js-next-meetings-container">
```
Comment (Norbiros) [inline gwardia/templates/gwardia/index.html]:
```suggestion
        <div class="w-1/2" id="js-next-meetings-container">
```

File: gwardia/templates/gwardia/index.html [hunk 7]
```diff
@@ -2,27 +2,30 @@
...
+            {% list_events upcoming_meetings "Nadchodzące" %}
+        </div>
+        <div class="w-1/2 hidden" id="js-archival-meetings-container">
```
Comment (Norbiros) [inline gwardia/templates/gwardia/index.html]: You could move `w-1/2` to new dive above those 2 and change it so it looks better on mobile devices (for example use `3/4`)

File: gwardia/templates/gwardia/list_events.html [hunk 8]
```diff
@@ -0,0 +1,23 @@
+<h2 class="text-2xl text-white text-center font-bold mt-8 mb-4">{{ title }} spotkania</h2>
+<ul>
+    {% if events %}
+        {% for event in events %}
+            <li class="js-meeting-container flex flex-col items-center border-solid border-gray-500 border-2 p-5 w-full mb-1 rounded-2xl cursor-pointer{% if event.is_archival %}bg-orange-500 bg-opacity-25{% endif %}">
```
Comment (Norbiros) [inline gwardia/templates/gwardia/list_events.html]:
```suggestion
            <li class="js-meeting-container flex flex-col items-center border-solid border-gray-500 border-2 p-5 w-full mb-1 rounded-2xl cursor-pointer {% if event.is_archival %}bg-orange-500 bg-opacity-25{% endif %}">
```

File: gwardia/templates/gwardia/list_events.html [hunk 9]
```diff
@@ -0,0 +1,23 @@
...
+            {% if event.is_archival %}
```
Comment (Norbiros) [inline gwardia/templates/gwardia/list_events.html]: Indent. Move whole `if` by 1

**Review by Norbiros — CHANGES_REQUESTED** [review]

File: gwardia/templates/gwardia/list_events.html [hunk 10]
```diff
@@ -0,0 +1,23 @@
...
+        {% for event in events %}
+            <li class="js-meeting-container flex flex-col items-center border-solid border-gray-500 border-2 p-5 w-full mb-1 rounded-2xl cursor-pointer {% if event.is_archival %}bg-orange-500 bg-opacity-25{% endif %}">
+                {% if event.is_archival %}
+                    <p class="text-orange-300 text-right w-full ">Archiwalne</p>
+                {% endif %}
+                <p class="w-full">
+                    <span class="break-all">{{ event.title }}</span>
+                    <span class="float-right">{{ event.date }}</span>
+                </p>
+                <p class="js-meeting-description border-gray-500 p-5 w-full break-word border-t-2 hidden">
```
Comment (Norbiros) [inline gwardia/templates/gwardia/list_events.html]: Remove this padding (if you really want just leave padding top)

File: gwardia/templates/gwardia/list_events.html [hunk 11]
```diff
@@ -0,0 +1,23 @@
+<h2 class="text-2xl text-white text-center font-bold mt-8 mb-4">{{ title }} spotkania</h2>
+<ul>
+    {% if events %}
+        {% for event in events %}
+            <li class="js-meeting-container flex flex-col items-center border-solid border-gray-500 border-2 p-5 w-full mb-1 rounded-2xl cursor-pointer {% if event.is_archival %}bg-orange-500 bg-opacity-25{% endif %}">
+                {% if event.is_archival %}
+                    <p class="text-orange-300 text-right w-full ">Archiwalne</p>
+                {% endif %}
+                <p class="w-full">
+                    <span class="break-all">{{ event.title }}</span>
```
Comment (Norbiros) [inline gwardia/templates/gwardia/list_events.html]: Maybe bold it?

Comment (Norbiros) [inline gwardia/templates/gwardia/list_events.html]:
```suggestion
                    <span class="break-words">{{ event.title }}</span>
```

File: gwardia/templates/gwardia/list_events.html (hunk 10 above)
Comment (Norbiros) [inline gwardia/templates/gwardia/list_events.html]:
```suggestion
                <p class="js-meeting-description border-gray-500 p-5 w-full break-words border-t-2 hidden">
```

**Review by Norbiros — CHANGES_REQUESTED** [review]
Also use `ruff format` (and fix yours pre-commit hook uwu)

File: gwardia/templates/gwardia/list_events.html (hunk 11 above)
Comment (Kubaryt) [reply gwardia/templates/gwardia/list_events.html:10]: image

**Review by Norbiros — APPROVED** [review]

**Pull request merged**